src/components/home/PostList/PostList.tsx

```diff
@@ -13,14 +14,18 @@ interface Post {
 interface PostListProps {
   posts: Post[];
   handleBookmarkClick: (postId: number) => void;
+  $isMypage: boolean;
 }
 
 const PostList: React.FC<PostListProps> = ({ posts, handleBookmarkClick }) => {
+  const location = useLocation();
+  const isMypage = location.pathname === '/mypage';
+
   return (
-    <S.PostList>
+    <S.PostList $isMypage={isMypage}>
       {posts.map((post) => (
         <S.PostItem key={post.id}>
```

Comment:
```suggestion
        const navigate = useNavigate();
        ....
        <S.PostItem onClick={navigate(`/mypage/${post.id}`} key={post.id}>
```
위와 같은 useNavigate 방식 사용하거나 PostItem을 `<a>` 태그로 바꾸고, `href` 속성 사용해서 해당 게시물로의 라우팅 미리 구현 해놓으면 좋을 것 같아요!